### PR TITLE
VBLOCKS-1438: refactor - remove rudimentary auth

### PR DESCRIPTION
**Pull request by kpchoy**
[VBLOCKS-1438](https://issues.corp.twilio.com/browse/VBLOCKS-1438)

Remove existing rudimentary auth

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by charliesantos — CHANGES_REQUESTED** [review]
@kpchoy I see a lot of changes updating single quotes to double quotes. Is this from your local linter? Please change it back first then I'll continue to review.

**Comment by kpchoy** [comment]
@charliesantos sorry about that, needed to add a prettierrc.js to server folder

File: server/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  bracketSameLine: true,
+  singleQuote: true,
+  trailingComma: "all",
```
Comment (charliesantos) [inline server/.prettierrc.js:4]:
```suggestion
  trailingComma: 'all',
```

Comment (mhuynh5757) [inline server/.prettierrc.js:4]:
```suggestion
  trailingComma: 'all',
```
Super small nitpick, just to match the `prettier` config from the `app/` folder.

**Review by charliesantos — CHANGES_REQUESTED** [review]
I still see references of auth related code in the following areas. Do we need to remove these or are we thinking of using them later? @kpchoy @mhuynh5757 
- app/src/store/user.ts and store.test.ts
- token.test.ts
- twiml.test.ts
- server.test.ts

File: app/__tests__/store.test.ts [hunk 2]
```diff
@@ -38,12 +38,6 @@ it('works good', async () => {
   console.log(app.store.getState());
 
   fetchMock.mockResolvedValueOnce({ status: 200 });
```
Comment (mhuynh5757) [inline app/__tests__/store.test.ts:40]:
```suggestion
  
```
We shouldn't need this line or the following `console.log` anymore.

File: server/src/routes/token.ts [hunk 3]
```diff
@@ -1,27 +1,19 @@
...
-      res.status(403).send('Unauthenticated request.');
-      return;
-    }
-
     const accessToken = new AccessToken(
       twilioCredentials.ACCOUNT_SID,
       twilioCredentials.API_KEY_SID,
       twilioCredentials.API_KEY_SECRET,
-      {
-        identity: authentication.username,
```
Comment (mhuynh5757) [inline server/src/routes/token.ts:23]: Note for future reference, we need this for incoming but are removing it presently just for finishing outgoing.

**Review by mhuynh5757 — APPROVED** [review]
Looking good!

**Comment by mhuynh5757** [comment]
> I still see references of auth related code in the following areas. Do we need to remove these or are we thinking of using them later? @kpchoy @mhuynh5757
> 
> * app/src/store/user.ts and store.test.ts
> * token.test.ts
> * twiml.test.ts
> * server.test.ts

For the test stuff, I think we can revisit when we do test tooling.

File: server/src/routes/token.ts (hunk 3 above)
Comment (kpchoy) [reply server/src/routes/token.ts:23]: Should I just comment it out and leave a note here @mhuynh5757 ?

Comment (mhuynh5757) [reply server/src/routes/token.ts:23]: Nah, I think it's ok to remove it. We don't know how the final API will end up looking, i.e. what if we have a different route for getting tokens for incoming calls.

**Review by charliesantos — APPROVED** [review]
Ok to merge once @mhuynh5757 's comments are addressed.

**Pull request merged**